### DISCUSSION TITLE
How to resolve tilelink bus errors？

### DISCUSSION THREAD

**Discussion by Balle0500** [polarfire-soc/polarfire-soc-documentation#406] [HSS and U-Boot]
May I ask how to resolve tilelink bus errors？
<img width="533" alt="Snipaste_2024-04-01_15-57-27" src="https://github.com/polarfire-soc/polarfire-soc-documentation/assets/155931587/3443c053-3bb6-4bc5-9dc0-ef6c4fc52b5d">

**Comment by hughbreslin** [comment]
This usually indicates a core has attempted to access a memory location that doesn't exist, something has gone wrong leading to the bus error instead of the bus error being an issue. I think you need to look at your other application code to see if its accessing a memory location it shouldn't
